ui/raidboss/data/04-sb/alliance/orbonne_monastery.js

```diff
@@ -1337,25 +1388,25 @@
         'Ephemeral Knight': '환영 기사',
         'Halidom': '모조 성역',
         'Harpy': '하피',
-        'I see it now': 'I see it now', // FIXME
+        'I see it now': '물병이여,  곧장 진격하여',
         'Icewolf': '빙랑',
         'Iron Construct': '노동형 철거인',
         'Mustadio': '기공사 무스타디오',
         'Ramza': '용사 람자',
         'Ruination': '루인 계열 취약',
         'Shield Knight': '방패의 기사',
         'Sword Knight': '검의 기사',
-        'The Crystalline Gaol will be sealed off': 'The Crystalline Gaol will be sealed off', // FIXME
-        'The Realm of the Machinists will be sealed off': 'The Realm of the Machinists will be sealed off', // FIXME
-        'The Realm of the Templars will be sealed off': 'The Realm of the Templars will be sealed off', // FIXME
-        'The Realm of the Thunder God will be sealed off': 'The Realm of the Thunder God will be sealed off', // FIXME
+        'The Crystalline Gaol': '수정 감옥',
```

Comment:
I don't understand how this works.  I think you need to replace the will be sealed off text as well.

Reply:
I think https://github.com/quisquous/cactbot/blob/0ece5b9e6839bcfc3188e7da4f7d6d4f2bc44329/resources/lang/ko.js#L85-L87
this code replaces `will be sealed off` things to Korean.

Reply:
https://github.com/quisquous/cactbot/blob/0ece5b9e6839bcfc3188e7da4f7d6d4f2bc44329/ui/raidboss/data/05-shb/dungeon/akadaemia_anyder.js#L512-L514
These are place names, didn't use `will be sealed off` things. It worked well.
https://github.com/quisquous/cactbot/blob/0ece5b9e6839bcfc3188e7da4f7d6d4f2bc44329/ui/raidboss/data/05-shb/dungeon/akadaemia_anyder.txt#L16
Timeline has `will be sealed off` things, but worked well.

Reply:
`areaSealRegex` is only used in `ui/dps/xephero/dps_phase_tracker.js` at the moment (based on `git grep areaSealRegex`.  It does not automatically replace things here.

If you have the replacement as you do above, it will be a replacement that looks like `00:0839:수생 생물 창조장 will be sealed off`, when your log probably looks like `00:0839:15초 후에 수생 생물 창조장`.  I think the replacement should be `'Ichthyology will be sealed off': '\\y{Float}초 후에 수생 생물 창조장` or something along those lines?

I think it doesn't break because there's also a sync to the first ability in the timeline, e.g. `103.0 "--sync--" sync /:Cladoselache:368:/ window 103,0`.

Reply:
I think adding `' will be sealed off': '(이|가) 봉쇄됩니다\.',` instead of replacing each of them is better. So, I did it.

Reply:
Hmmmm... --reset-- works without `'is no longer sealed': '의 봉쇄가 해제되었습니다',` replacement.
I think something is replacing it.

Reply:
See #1032 about more.